### PR TITLE
First attempt at adding props to RNAseqTemplate

### PR DESCRIPTION
**Pull request by aditya-nath-sage**
Added requested fields to Data.yml in RNASeqTemplate. 

Do I need to update any testing resources? Was this the correct Assay to update?

### CODE REVIEW

File: modules/Template/Data.yaml [hunk 1]
```diff
@@ -178,6 +178,10 @@ classes:
         assay:
         - RNA-seq
         - lncRNA-seq
+        - genePerturbed
```
Comment (anngvu) [inline modules/Template/Data.yaml:181]: Hey Aditya, this is the correct template, but in our walkthrough, a similar update was made by adding attributes to under `slots:`. Remember that this assay annotation is meant for mapping, and not actually used.

**Review by anngvu — CHANGES_REQUESTED** [review]
See comment

File: modules/Template/Data.yaml (hunk 1 above)
Comment (aditya-nath-sage) [reply modules/Template/Data.yaml:181]: Ahh got it thank you!

**Review by anngvu — COMMENTED** [review]
(Also, might want to reset and use more recent updates from main to avoid the conflict.)

**Review by anngvu — APPROVED** [review]
Good job @aditya-nath-sage! Note: the two failing tests are expected due to some schematic issues. Merging.

**Pull request merged**